doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -6,57 +6,24 @@
 
 ### Features
 
-<!--
-    NOTE: Add new entries sorted by issue number to minimize the possibility of
-    conflicts when merging.
--->
+Create a new CLI for iceoryx2 `iox2-config`
```

Comment:
Please summarize everything into one sentence and put everything in one bullet point as you did in line 15. If someone is interested in the details, they can follow the provided issue link.